### PR TITLE
Fixing typing of fulfillment simulation body request

### PR DESCRIPTION
**Pull request by fabianaferreira**
Fixes typings that we wrongly defined.

### CODE REVIEW

File: node/typings/fullfilmentSimulation.ts [hunk 1]
```diff
@@ -100,23 +102,34 @@ interface Address {
 export interface SimulationInput {
   postalCode: string
   geoCoordinates: string[]
-  country: string
-  items: CartItem[]
+  country: string | null
+  items: Item[]
+  priceTables: any[]
+  marketkingData: any
```
Comment (attaalan) [inline node/typings/fullfilmentSimulation.ts]: Should be `marketingData` instead?

Comment (fabianaferreira) [reply node/typings/fullfilmentSimulation.ts]: :confused-parrot: :nazare:

File: node/typings/fullfilmentSimulation.ts [hunk 2]
```diff
@@ -100,23 +102,34 @@ interface Address {
...
+  street: string
+  isFOB: boolean
+  selectedAddresses: any[]
+  logisticsInfo: LogisticsInfo[]
+}
+
+interface Item {
   id: string
   seller: string
-  sku: string
```
Comment (gris) [inline node/typings/fullfilmentSimulation.ts:110]: Is this removed data coming?

Comment (fabianaferreira) [reply node/typings/fullfilmentSimulation.ts:110]: As far as I know, it's not. We tested on `famstore`, to make sure of which are the fields that are actually coming.

**Review by gris — APPROVED** [review]

**Pull request merged**